src/test/scala/com/mwz/sonar/scala/scoverage/ScoverageReportParserSpec.scala

```diff
@@ -27,7 +27,7 @@ import org.scalatest.{FlatSpec, Inside, LoneElement, Matchers}
 class ScoverageReportParserSpec extends FlatSpec with Inside with LoneElement with Matchers {
   val modulePath: Path = Paths.get("")
```

Comment:
I would remove the `: Path` too.
Also, there are a few more changes I made to this spec.
Like removing unused traits\imports.

Reply:
Ok, done.